lib/cli-engine.js

```diff
@@ -302,6 +305,17 @@ function CLIEngine(options) {
      */
     this.options = assign(Object.create(defaultOptions), options || {});
 
+    /**
+     * cache used to not operate on files that haven't changed since last successful
+     * execution (e.g. file passed with no errors and no warnings
+     * @type {Object}
+     */
+    this.fileCache = fileEntryCache.create(path.resolve(this.options.cacheFile));
```

Comment:
Just want to double-check: is there any reason a user of `CLIEngine` would want to access the cache? If not, it should be prepended with an `_` to indicate it's private.